client/algod/models/models.go

```diff
@@ -145,6 +145,19 @@ type AssetParams struct {
 	// required: false
 	AssetName string `json:"assetname"`
 
+	// URL specifies a URL where more information about the asset can be
+	// retrieved
+	//
+	// required: false
+	URL string `json:"url"`
+
+	// MetadataHash specifies a commitment to some unspecified asset
+	// metadata. The format of this metadata is up to the application.
+	//
+	// required: false
+	// swagger:strfmt byte
+	MetadataHash []byte `json:"metadatahash"`
```

Comment:
Note that these have changed a bit (nothing too extreme). refer to https://github.com/algorand/go-algorand/blob/master/data/basics/userBalance.go#L206